serverless/src/functions/mints/handler.ts

```diff
@@ -6,31 +6,192 @@ import {
 import { formatJSONResponse } from '@libs/api-gateway';
 
 import { v4 } from 'uuid';
+import { initPrisma, prisma } from '@libs/prisma';
+import { account, nfaContract, web3 } from '@libs/nfa-contract';
 
 export const submitMintInfo = async (
   event: APIGatewayEvent
   ///context: APIGatewayEventRequestContext
 ): Promise<APIGatewayProxyResult> => {
   try {
+    if (event.body === null) {
+      return formatJSONResponse({
+        status: 422,
+        message: 'Required parameters were not passed.',
+      });
+    }
     const id = v4();
 
     /**if (!verifyAlchemySig(event.headers.xalchemywork)) {
         throw new Error('Invalid sig');
       }**/
 
-    if (event.body == undefined) {
-      throw new Error('Undefined data');
-    }
+    let topics = [
+      JSON.parse(event.body).event.data.block.logs[1].topics[1],
+      JSON.parse(event.body).event.data.block.logs[1].topics[2],
+      JSON.parse(event.body).event.data.block.logs[1].topics[3],
+    ];
+    const hexCalldata = JSON.parse(event.body).event.data.block.logs[1].data;
```

Comment:
```suggestion
const eventBody = JSON.parse(event.body);
const topics = eventBody.event.data.block.logs[1].slice(1, 3);
const hexCalldata = eventBody.event.data.block.logs[1].data;
```

Reply:
Good change. I had forgotten to change topics lol